frontend/package.json

```diff
@@ -60,8 +60,10 @@
     "@types/redux-logger": "3.0.7",
     "@types/redux-mock-store": "1.0.2",
     "axios-mock-adapter": "1.17.0",
+    "canvas": "^2.6.1",
```

Comment:
I keep getting errors locally when `npm test`.  Adding it seems to solve some of the errors.  Not sure it's required though.

Reply:
I would prefer to dig a bit deeper into why this is required before adding it. Which specific test is failing?

Reply:
I don't actually know for certain, as I'm having a hard time reading it all.
![image](https://user-images.githubusercontent.com/3180256/79783492-995f9d00-82f5-11ea-805c-60a2221ccfb6.png)

Reply:
That is because you are not running Jest in verbose mode. See my comment on RC:
```sh
npm test -- --verbose
```

Reply:
If you do end up adding this, make sure to remove the ^

Reply:
fixed